### DISCUSSION TITLE
Why is there only "Pre-release" versions, and no more "official release" since a quite long time ?

### DISCUSSION THREAD

**Discussion by LaurentGrenet** [Open-Shell/Open-Shell-Menu#1609] [General]
In other words, is it safe, or not  recommended, to install the last pre-release over the last "official" release, that is more than one year old, and followed by 17 "pre-release" versions ?

**Comment by ge0rdi** [comment]
You can try them for yourself. In case of any issue we'd appreciate a feedback. And you can still return to official one at any time.
The more people will try pre-release the more confident we can be to release it as official.

Though I plan to make 4.4.190 official one soon.
